lms/djangoapps/discussion/views.py

```diff
@@ -581,6 +581,12 @@ def user_profile(request, course_key, user_id):
             })
         else:
             tab_view = CourseTabView()
+
+            # To avoid mathjax loading from 'mathjax_include.html'
```

Comment:
max line length for comments in : 
https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html
https://openedx.atlassian.net/wiki/spaces/EDUCATOR/pages/251528559/Coding+Practices

Reply:
the line length for comments is being respected (<79).